src/Selenium2Library/keywords/_element.py

```diff
@@ -1,14 +1,32 @@
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.remote.webelement import WebElement
 from Selenium2Library import utils
 from Selenium2Library.locators import ElementFinder
 from keywordgroup import KeywordGroup
 
+try:
+    basestring  # attempt to evaluate basestring
+    def isstr(s):
+        return isinstance(s, basestring)
+except NameError:
+    def isstr(s):
+        return isinstance(s, str)
+
 class _ElementKeywords(KeywordGroup):
 
     def __init__(self):
         self._element_finder = ElementFinder()
 
+    # Public, get element(s)
+
+    def get_webelements(self, locator):
```

Comment:
@ombre42 and I discussed this over on #315. 'Get Elements' was my original name but changed it because it conflicted with the XML Library's keyword of the same name. So we settled on Get WebElements.  I also had a 'Get WebElement' (singular) keyword in there but removed it for some reason. I was doing some recent testing with this and saw a possible need for getting a single web element.